reference/5.1/Microsoft.PowerShell.LocalAccounts/New-LocalUser.md

```diff
@@ -287,7 +285,7 @@ This object provides information about the user account.
 - Azure Active Directory group
 - Microsoft Account
 
-**PrincipalSource** is supported only by Windows 10, Windows Server 2016, and later versions of the Windows operating system. For earlier versions, the property is blank.
+**PrincipalSource** is supported only by Windows 10, Windows Server 2016, and later versions of the Windows operating system. For earlier versions, the property is blank. As of Windows 10 1909 with PowerShell 5.1 the **PrincipalSource** property is no longer available.
```

Comment:
I don't believe this is the case. If you run `Get-LocalUser -Name Administrator | Format-List *` you will see the **PrincipalSource** Attribute listed. 

I have 1909 running currently. If you run the above command, do you see something different?

Reply:
That may be true, but there is no -PrincipalSource parameter for the New-LocalUser cmdlet.  Maybe the wording needs to be clarified to avoid confusion?  Idk.  Just trying to help make it easier to understand.

Reply:
You are right, there isn't a PrincipalSource param, but I don't think there ever has been. It states in the notes that it's a property, not a parameter. I do think the formatting does make it a little more difficult to parse out the meaning correctly though.

What if it looked a something a little more like this: 
```
## NOTES

- A user name cannot be identical to any other user name or group name on the computer. A user name
  cannot consist only of periods (.) or spaces. A user name can contain up to 20 uppercase
  characters or lowercase characters. A user name cannot contain the following characters:

`" / \ \[ \] : ; | = , + * ? \< \> @`

- A password can contain up to 127 characters.
- The **PrincipalSource** property is a property on **LocalUser**, **LocalGroup**, and
  **LocalPrincipal** objects that describes the source of the object. The possible sources are as
  follows:

  - Local
  - Active Directory
  - Azure Active Directory group
  - Microsoft Account

> [!NOTE]
> **PrincipalSource** is supported only by Windows 10, Windows Server 2016, and later versions of the
> Windows operating system. For earlier versions, the property is blank.
```

Reply:
Perfect!  Thank you!